spec/support/repo/Gemfile

```diff
@@ -14,8 +14,9 @@ gem("psych")
 # ensure that we don't fail.
 gem("extras")
 # Needed for DSL generation tests. The simplest
-# gem that we have a DSL generator for.
+# gems that we have a DSL generator for.
 gem("smart_properties")
+gem("sidekiq")
```

Comment:
Added sidekiq so we have more than one type of DSL generator and can accurately test exclusion.